### PR TITLE
SWBD Result Update

### PR DESCRIPTION
**Pull request by roshansh-cmu**
Updates:

1. Collecting Results from .sys files rather than result.txt wherever available ( STM based Scoring) in scripts/utils/show_asr_result.sh 
2. Minor update in local/score.sh to make datadir editable for scoring rt03, in local/data.sh to use separate Fisher path
3. Added AM and LM confs for 7.2/14.6 and 6.8/14.0 from Conformer without External LM and Conformer with Transformer LM
Uses fbank-pitch features like ESP1: haven't tried raw yet

### CODE REVIEW

File: egs2/swbd/asr1/local/data.sh [hunk 1]
```diff
@@ -38,7 +38,7 @@ fi
   # SWBD: LDC97S62 LDC2002S09 LDC2002T43 LDC2004T19 LDC2005T19 LDC2004S13 LDC2005S13
 swbd1_dir=${SWBD}/LDC97S62
 eval2000_dir="${SWBD}/LDC2002S09/hub5e_00 ${SWBD}/LDC2002T43"
-fisher_dir="${SWBD}/LDC2004T19 ${SWBD}/LDC2005T19 ${SWBD}/LDC2004S13 ${SWBD}/LDC2005S13"
+fisher_dir="${FISHER}/LDC2004T19 ${FISHER}/LDC2005T19 ${FISHER}/LDC2004S13 ${FISHER}/LDC2005S13"
```
Comment (sw005320) [inline egs2/swbd/asr1/local/data.sh:41]: Why did you change it from `SWBD` to `FISHER`?
You can keep it as `SWBD` (also in `db.sh`).

**Review by sw005320 — COMMENTED** [review]
Many thanks, @roshansh-cmu!!
- [ ] Can you add an ASR result and emphasize where did you change from @yuekaizhang's result?
- [ ] Can you make your config as default by changing the symlink of https://github.com/espnet/espnet/blob/master/egs2/swbd/asr1/conf/train_asr.yaml and https://github.com/espnet/espnet/blob/master/egs2/swbd/asr1/conf/train_lm.yaml?

**Comment by sw005320** [comment]
@yuekaizhang, could you also review this PR?

**Review by yuekaizhang — APPROVED** [review]

**Comment by roshansh-cmu** [comment]
- Looks like the E2E ASR models have very similar config- the main difference is fbank-pitch versus raw, higher LR and longer training without early stopping for the E2E ASR model. 
- For the LM I used a smaller model with lower LR, and it gets perplexity is 36.66 v/s 51.07 from Yuekai's model.
Made all changes, will make final push soon

**Comment by sw005320** [comment]
Please fix some CI errors (https://github.com/espnet/espnet/pull/3308/checks?check_run_id=2875618722#step:7:96)

**Comment by sw005320** [comment]
Can you also add a short description of swbd in https://github.com/espnet/espnet/blob/master/egs2/README.md?

File: egs2/TEMPLATE/asr1/scripts/utils/show_asr_result.sh [hunk 2]
```diff
@@ -47,18 +47,25 @@ while IFS= read -r expdir; do
...
-            if ls "${expdir}"/*/*/score_${type}/result.txt &> /dev/null; then
+            
+		if ls "${expdir}"/*/*/score_${type}/result.txt &> /dev/null; then
                 cat << EOF
 ### ${type^^}
 
-|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
-|---|---|---|---|---|---|---|---|---|
+#|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+#|---|---|---|---|---|---|---|---|---|
```
Comment (sw005320) [inline egs2/TEMPLATE/asr1/scripts/utils/show_asr_result.sh:56]:
```suggestion
|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
|---|---|---|---|---|---|---|---|---|
```
Why did you comment out these lines?

File: egs2/TEMPLATE/asr1/scripts/utils/show_asr_result.sh [hunk 3]
```diff
@@ -47,18 +47,25 @@ while IFS= read -r expdir; do
...
 EOF
-                grep -H -e Avg "${expdir}"/*/*/score_${type}/result.txt \
-                    | sed -e "s#${expdir}/\([^/]*/[^/]*\)/score_${type}/result.txt:#|\1#g" \
-                    | sed -e 's#Sum/Avg##g' | tr '|' ' ' | tr -s ' ' '|'
-                echo
-            fi
+			if  [[ $type == "wer" ]] && [[ -n $(ls ${expdir}/*/*/score_wer/scoring/*.filt.sys) ]] ; then
```
Comment (sw005320) [inline egs2/TEMPLATE/asr1/scripts/utils/show_asr_result.sh:58]: Maybe you can add a comment for this case?

**Review by sw005320 — APPROVED** [review]

**Comment by roshansh-cmu** [comment]
All updates done from my side. Let me know if anything else is needed

**Comment by sw005320** [comment]
LGTM!
Thanks, @roshansh-cmu!
I'll merge this PR after the CI check.

**Comment by sw005320** [comment]
Can you register your model?
See https://github.com/espnet/espnet_model_zoo#register-your-model

**Comment by roshansh-cmu** [comment]
Done: https://github.com/espnet/espnet_model_zoo/pull/37
